source/styleguide/molecules/SgPageShell/SgPageShell.jsx

```diff
@@ -86,13 +78,7 @@ export const SgPageShell = (props) => {
 
   return (
     <div className={classStack} {...attrs}>
-      <Scrollbars
```

Comment:
Lets leave the custom scroll bars please. This is for Windows users like me who don't get nice fancy thin OSX scroll bars :D